pkg/storage/bloom/v1/block.go

```diff
@@ -144,14 +144,12 @@ func (bq *BlockQuerier) Seek(fp model.Fingerprint) error {
 func (bq *BlockQuerier) Next() bool {
 	for bq.series.Next() {
 		series := bq.series.At()
-		bq.blooms.Seek(series.Offset)
+		if ok := bq.blooms.LoadOffset(series.Offset); !ok {
```

Comment:
I think LoadOffset should return an error. Then we should check here if the error is due to the page being too big in which case we can safely continue. Otherwise, we wont be able to error when there is. for example, an IO error.